pkg/vminfo/vminfo_test.go

```diff
@@ -121,29 +120,62 @@ func createSuccessfulResults(source queue.Source, stop chan struct{}) {
 func hostChecker(t *testing.T) (*Checker, []*flatrpc.FileInfo) {
 	cfg := testConfig(t, runtime.GOOS, runtime.GOARCH)
 	checker := New(cfg)
-	files := host.ReadFiles(checker.RequiredFiles())
+	files := readFiles(checker.RequiredFiles())
 	return checker, files
 }
 
-func testConfig(t *testing.T, OS, arch string) *mgrconfig.Config {
+func testConfig(t *testing.T, OS, arch string) *Config {
 	target, err := prog.GetTarget(OS, arch)
 	if err != nil {
 		t.Fatal(err)
 	}
-	cfg := &mgrconfig.Config{
-		Sandbox: ipc.FlagsToSandbox(0),
-		Derived: mgrconfig.Derived{
-			TargetOS:     OS,
-			TargetArch:   arch,
-			TargetVMArch: arch,
-			Target:       target,
-			SysTarget:    targets.Get(OS, arch),
-		},
-	}
+	var syscalls []int
 	for id := range target.Syscalls {
 		if !target.Syscalls[id].Attrs.Disabled {
-			cfg.Syscalls = append(cfg.Syscalls, id)
+			syscalls = append(syscalls, id)
+		}
+	}
+	return &Config{
+		Target:   target,
+		Features: flatrpc.AllFeatures,
+		Sandbox:  flatrpc.ExecEnvSandboxNone,
+		Syscalls: syscalls,
+	}
+}
+
+func readFiles(files []string) []*flatrpc.FileInfo {
+	var res []*flatrpc.FileInfo
+	for _, glob := range files {
+		glob = filepath.FromSlash(glob)
+		if !strings.Contains(glob, "*") {
+			res = append(res, readFile(glob))
+			continue
+		}
+		matches, err := filepath.Glob(glob)
+		if err != nil {
+			res = append(res, &flatrpc.FileInfo{
+				Name:  glob,
+				Error: err.Error(),
+			})
+			continue
 		}
+		for _, file := range matches {
+			res = append(res, readFile(file))
+		}
+	}
+	return res
+}
+
+func readFile(file string) *flatrpc.FileInfo {
+	data, err := os.ReadFile(file)
```

Comment:
Will it be reading the actual host files during the test? If yes, don't we rather want to have a more contained environment in our tests?

Reply:
That's what we do now, I just moved this code from deleted pkg/host.
The tests that read host files don't do any checks, we just ensure that the code does not panic/error (host is one of possible Linux'es) and print results for manual inspection.